src/components/data-processing/GenesVsUMIs/CalculationConfig.jsx

```diff
@@ -38,28 +74,66 @@ const GenesVsUMIsConfig = (props) => {
         <Select
           value={config.regressionType}
           onChange={(val) => updateSettings({ regressionType: val })}
-          disabled={disabled}
         >
           <Option value='linear'>linear</Option>
           <Option value='spline'>spline</Option>
         </Select>
       </Form.Item>
-      <Form.Item label='p-level cut-off:'>
+      <Form.Item label='Prediction interval'>
+
+        <Tooltip title=' Regression of feature counts (genes) vs UMI counts (molecules) is performed for all cells in order to detect outliers.
+            The ‘prediction interval’ is the stringency for defining outliers: it sets the prediction intervals calculated by the R `predict`
+            where `level = prediction interval`. Prediction intervals represent the likelihood that the predicted value will be between the upper and
+            lower limits of the prediction interval. Prediction intervals are similar to confidence intervals, but on top of the sampling uncertainty,
+            they also express uncertainty around a single value. They must account for the uncertainty in estimating the population mean,
+            plus the random variation of the individual values. Higher prediction interval means higher probability of the value to be inside the range.
+            Consequently, the size of the interval will be wider. The higher the prediction level, the less stringent we are when filtering the cells.
+            Conversely, the lower the prediction level, the more stringent we are, and we exclude more cells that are far from the behaviour of the
+            relationship between the number of genes and the number of UMIs/molecules.'
+        >
+          <InfoCircleOutlined />
+        </Tooltip>
+
+        <Slider
+          value={newConfig.predictionInterval}
+          min={0}
+          max={0.99}
+          step={0.01}
+          disabled={disabled}
+          onChange={(val) => handleChange({ predictionInterval: val })}
+        />
+
+        <Radio.Group
+          value={config.predictionInterval}
+          onChange={(val) => updateSettings({ predictionInterval: val.target.value })}
+          disabled={disabled}
+        >
+          <Space direction='vertical'>
+            <Space direction='horizontal'>
+              <InputNumber
+                value={newConfig.predictionInterval || 1 - config.regressionTypeSettings[config.regressionType]['p.level']}
```

Comment:
`1 - config.regressionTypeSettings[config.regressionType]['p.level'] ` is the default plevel? Can you make a new variable for this (e.g. defaultPLevel), and use that variable in here? It'll increase readibility